presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -11,12 +11,17 @@ The Iceberg connector allows querying data stored in Iceberg tables.
 
     It is recommended to use Iceberg 0.9.0 or later.
```

Comment:
Yes, that's right. Users will not have any control over this from the Presto engine side. And current Iceberg version in Presto is `1.3.1`. As I checked Presto iceberg connector started from `0.9.0` this might have been highlighted from there for other engines.